topcoder-platform/tc-online-review
# Disable editing challenge name and type + remove Distributions management

This PR addresses the following items:
1. Remove the ability to edit the challenge name
2. Remove the ability to change the “Type” (Application|Component|etc)
3. Remove the code and UI that is available when you click “Manage Project” for uploading distributions.  This is only available when you are managing a Component Design/Development challenges.

# Verification for items 1 and 2:
- Login to Online Review with a user with 'manager' role and open any challenge details page.
- Click on 'Edit Project' on the bottom right of the page
- Under the 'Project Details' section, the 'Name:' and 'Type:' fields should not be editable as shown below:
![Challenge-Name-AndType](https://user-images.githubusercontent.com/5437419/80128821-c0cc9900-8585-11ea-9129-a8e2d701d867.png)

# Verification for item 3:
- Open any Component Design or Component Development challenge details page.
- Click on 'Manage Project' button on the bottom of the page, the 'Distributions' tab should not be shown
![Distributions](https://user-images.githubusercontent.com/5437419/80130184-bca17b00-8587-11ea-843c-5ec084e3c3c0.png)
